### PR TITLE
Fix ci docker image name issue

### PR DESCRIPTION
**Pull request by huikang**
Signed-off-by: Hui Kang kangh@us.ibm.com

### CODE REVIEW

**Comment by huikang** [comment]
Fix #74

File: ci/ansible/all.yml [hunk 1]
```diff
@@ -9,10 +9,10 @@ deploy_user: "root"
 # Docker options
 ###################
 
-ovn_db_image: "huikang/ovn-scale-test-ovn"
-ovn_chassis_image: "huikang/ovn-scale-test-ovn"
+ovn_db_image: "ovn-scale-test-ovn"
```
Comment (mestery) [inline ci/ansible/all.yml:12]: I'd prefer if this PR had the changes to the above files separate, and a "git revert" of the commit I did here. Can you do that and then I will merge this?

**Comment by mestery** [comment]
I tested the same change and it worked, my only comment is that we should revert my commit as part of this PR (git revert 648cdac73ab608dc56fa3ed3faf0f514fdf8b348) and then make the changes to the two docker build files. This PR would then have two commits (the revert and your new changes). Can you do that please?

**Comment by huikang** [comment]
@mestery sure, I will separate them and close this one. Thanks for reviewing the PR.

**Comment by mestery** [comment]
@huikang I merged #78, can you re-spin this one now and then I'll merge this one to get the CI running again? Thanks!

**Comment by mestery** [comment]
Thanks @huikang! Once this passes travis-ci I'll merge it!

**Comment by huikang** [comment]
@mestery cool

**Pull request merged**